R/analysis_page.R

```diff
@@ -378,11 +376,59 @@ AnalysisPageServer <- function(id, data) {
                   timevar = "sens" , times = c(1,0) , v.names = c("n","true") ) 
       Y = Y[order(Y$id),]  
       Y$spec<- 1-Y$sens
+      MA_Y <- glmer(formula = cbind(  true , n - true ) ~ 0 + sens + spec + (0+sens + spec|study),
+                    data = Y , family = binomial , nAGQ=1 , verbose=0 )
+      return(MA_Y)
+    })
+    
+    #Model summary
+    model_summary <- reactive(summary(model_output()))
+    
+    #Check if the model converged
+    converged <- reactive(is.null(model_summary()$optinfo$conv$lme4$messages))
+    
+    #Add converged() reactive to the output list
+    output$converged <- reactive(converged())
+    outputOptions(x = output, name = "converged", suspendWhenHidden = FALSE)
+    
+    #If the model did not converge, display a warning
+    observe({
+      if (!converged()) {
+        shinyalert(
+          title = "WARNING - model failed to converge",
+          text = paste0("Most output will not be displayed. \n\n MetaDTA is not equipped for model diagnostics. You are advised to use MetaBayesDTA (https://www.gla.ac.uk/research/az/crsu/apps-materials-guidence/) or other software. \n\nError message: \n", model_summary()$optinfo$conv$lme4$messages),
```

Comment:
Can you make this a link please? Also, should it go straight to the app, or have you deliberately sent users to the website to show them all the other apps too?

Reply:
It's deliberate to send them to the "home page".